workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -54,6 +54,11 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                         new LogEvent("Cancelling connection attempt")
                             .WithField("WorkerType", workerType)
                             .WithField("Reason", "Required worker failed to connect"));
+#if UNITY_EDITOR
+                    logger.HandleLog(LogType.Warning,
+                        new LogEvent("A local runtime may not have been started. If not, you can start one from 'SpaitalOS -> Local launch' or by pressing Ctrl-L")                      
```

Comment:
That's not accurate on Mac, where it will be "Cmd" instead of "Ctrl".

Reply:
Ah, good point! I assume Cmd/Ctrl-L would probably be acceptable?